### PR TITLE
FIX natural order by (avoid 1,10,11,2,...)

### PR DESCRIPTION
**Pull request by hregis**

### CODE REVIEW

File: myaccount/tpl/instances.tpl.php [hunk 1]
```diff
@@ -46,7 +46,7 @@
 $sqlproducts.= " OR pa.restrict_domains LIKE '%,".$db->escape($domainname).",%'"; // can be the middle domain of [mydomain1.com,mydomain2.com,mydomain3.com]
 $sqlproducts.= " OR pa.restrict_domains LIKE '%,".$db->escape($domainname)."'"; // can be the last domain of [mydomain1.com,mydomain2.com]
 $sqlproducts.= ")";
-$sqlproducts.= " ORDER BY pe.position ASC";
```
Comment (eldy) [inline myaccount/tpl/instances.tpl.php:49]: Did the field pe.position is an integer into database or a string ?

Comment (hregis) [reply myaccount/tpl/instances.tpl.php:49]: @eldy this the new service extrafield, this is a varchar. it's better with an integer ?

Comment (hregis) [reply myaccount/tpl/instances.tpl.php]: @eldy ok, i change to use integer instead varchar for the extrafield "position"

**Comment by hregis** [comment]
@eldy ping

**Pull request merged**